source/develop/release-management/features/network/isolated-ports.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Isolated ports
+category: feature
+authors: dholler
+feature_name: Isolated ports
+feature_modules: engine,network
+feature_status: In Development
+---
+
+# Isolated ports
+
+## Summary
+
+This feature prevents communication of VMs and even visibility between VMs
+connected to the same logical network.
+
+The state of the feature is tracked in [[RFE] Private VLAN / port isolation
+][1]. Related patches can be found on [gerrit topic:isolated_port][2].
+
+## Owner
+
+*   Name: [Dominik Holler](https://github.com/dominikholler)
+
+*   Email: <dholler@redhat.com>
+
+
+## Detailed Description
+
+There are scenarios, where multiple VMs, e.g. owned by multiple customers, are
```

Comment:
I'm not a native English speaker but it looks like an over-use of commas. This is not just in the sentence but this one really feels "minced".

Reply:
I spited the sentence.